src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs

```diff
@@ -15,14 +20,68 @@ internal static class WorkloadCommandParser
 
         private static readonly Command Command = ConstructCommand();
 
+        public static readonly Option<bool> InfoOption = new Option<bool>("--info");
+
         public static Command GetCommand()
         {
+            Command.AddOption(InfoOption);
             return Command;
         }
 
+        internal static void ShowWorkloadsInfo(IWorkloadInfoHelper workloadInfoHelper = null, IReporter reporter = null)
+        {
+            workloadInfoHelper ??= new WorkloadInfoHelper();
+            IEnumerable<WorkloadId> installedList = workloadInfoHelper.InstalledSdkWorkloadIds;
+            InstalledWorkloadsCollection installedWorkloads = workloadInfoHelper.AddInstalledVsWorkloads(installedList);
+            reporter ??= Cli.Utils.Reporter.Output;
+
+            if(!installedList.Any())
+            {
+                reporter.WriteLine(CommonStrings.NoWorkloadsInstalledInfoWarning);
+                return;
+            }
+
+
+            foreach (var workload in installedWorkloads.AsEnumerable())
+            {
+                reporter.WriteLine("\n");
+                reporter.WriteLine(CommonStrings.WorkloadIdColumn + " : [" + workload.Key + "]");
+
+                reporter.Write(CommonStrings.WorkloadSourceColumn + ":");
+                reporter.WriteLine("\t" + workload.Value);
+
+                var workloadManifest = workloadInfoHelper.WorkloadResolver.GetManifestFromWorkload(new WorkloadId(workload.Key));
+                var workloadFeatureBand = new WorkloadManifestInfo(
+                    workloadManifest.Id,
+                    workloadManifest.Version,
+                    Path.GetDirectoryName(workloadManifest.ManifestPath)!).ManifestFeatureBand;
+
+                reporter.Write(CommonStrings.WorkloadManfiestVersionColumn + ":");
+                reporter.WriteLine("\t" + workloadManifest.Version + "/" + workloadFeatureBand);
+
+                reporter.Write(CommonStrings.WorkloadManifestPathColumn + ":");
```

Comment:
I'm a bit on the fence with the inline tabs. Can we use interpolated strings with column widths or maybe a `PrintableTable` without column headings, that way the formatting remains more consistent, especially if the actual spaces in tabs varies between platforms

Reply:
I don't think we should use a `printabletable`, I used that initially, but:
a) it doesn't match `dotnet --info` even if it matches `dotnet workload list`
b) the manifest path is so long that the table becomes unreadable unless the command line is full screen and on a huge monitor

Interpolated strings though I advocate for!

Reply:
Please see committed fixes.